features/google/src/views/overview/overview.jsx

```diff
@@ -20,8 +21,16 @@ const Overview = () => {
         refreshFiles();
         history.push("/import");
     }
+    useEffect(() => {
+        if (!account) return;
+        analyzeFile({
+            zipData: files[0],
```

Comment:
Elsewhere, we're preparing this for analyzing multiple files. I wonder how this could change in that case.

Reply:
maybe loop over the length of files and execute for each file?